src/ctia/stores/es/init.clj

```diff
@@ -155,15 +155,22 @@
                         :mappings mappings))
       (system-exit-error))))
 
+(defn ambiguous-indices
+  [existing index]
+  (let [existing-k (set (keys existing))
+        index-pattern (re-pattern (str "((partial|restored)-)?"
+                                       index
```

Comment:
Might want to quote this string: `(Pattern/quote index)`

https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html#quote-java.lang.String-